lib/sourceParams.js

```diff
@@ -72,6 +72,7 @@ module.exports = function (app) {
     shutdownTimeout: checkParams(params.shutdownTimeout, pkg.rooseveltConfig.shutdownTimeout, defaultConfig.shutdownTimeout),
     cores: checkParams(params.cores, pkg.rooseveltConfig.cores, defaultConfig.cores),
     cleanTimer: checkParams(params.cleanTimer, pkg.rooseveltConfig.cleanTimer, defaultConfig.cleanTimer),
+    reload: checkParams(params.reload, pkg.rooseveltConfig.reload, defaultConfig.reload),
```

Comment:
You probably want to make this name tool agnostic. Feel free to tell me what you want to change it to and I can make the change.

Reply:
Let's go with `frontendReload`. This distinguishes it from a backend reloader like `nodemon`, and we can draw that distinction in the README.

Reply:
Updated